src/App.js

```diff
@@ -155,6 +177,11 @@ export default class App extends React.Component<{}, AppState> {
         );
     }
 
+    componentDidMount() {
+        this.soundAnalyser.getMicrophones();
+        this.soundAnalyser.configureDetection({"cb3b7c5cefb4536a35cdb8ab6975a00705300108fe461fc31965de349f3722d7" : this.handleClap});
```

Comment:
We aren't using the discovered microphones. I'm thinking "cb3b7c5cefb4536a35cdb8ab6975a00705300108fe461fc31965de349f3722d7" is an id from your laptop?